AdyenComponents/Apple Pay/ApplePayNetworksProvider.swift

```diff
@@ -0,0 +1,19 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+import PassKit
+
+internal protocol ApplePayNetworksProviding {
+    func availableNetworks() -> [PKPaymentNetwork]
+}
+
+internal struct ApplePayNetworksProvider: ApplePayNetworksProviding {
```

Comment:
IMHO, this is overkill. Apple providing us with a list of networks. It would never be nil or empty; and if it stop providing a network (`mir` for example) - it is not like Adyen can force it to work 😅